packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -26,6 +26,13 @@ export 'package:flutter/services.dart' show TextSelectionDelegate;
 /// called.
 const Duration _kDragSelectionUpdateThrottle = Duration(milliseconds: 50);
 
+
+/// If the distance from the top is less than a certain value,
+/// the toolbar should be displayed below the input box.
+/// If don't do this, won't be able to properly display and interact toolbar at the top of some phones, such as iPhone X.
```

Comment:
This line should be broken so that it fits our 80 character limit.

Also, is it possible to reuse this constant so we don't have to duplicate it above?  If not, maybe it's possible to reuse the comment with our `@macro` syntax (see an example of this [here](https://github.com/flutter/flutter/blob/master/packages/flutter/lib/src/material/text_field.dart#L244)).

Reply:
Thanks Reply.
The current writing is the reference to the two writing styles([material/text_selection](https://github.com/flutter/flutter/blob/ed49122aa3a1c207614c438fdaad2eeec3e39595/packages/flutter/lib/src/material/text_selection.dart#L19) [cupertino/text_selection](https://github.com/flutter/flutter/blob/ed49122aa3a1c207614c438fdaad2eeec3e39595/packages/flutter/lib/src/cupertino/text_selection.dart#L19)),
Unfortunately, I have not found a suitable place yet to properly define this constant and reuse it.